apps/new-dealer/views/step.html

```diff
@@ -1,8 +1,5 @@
 {{<partials-page}}
   {{$page-content}}
-    {{#renew}}
-      {{> partials-important-message}}
-    {{/renew}}
     {{#fields}}
       {{#renderField}}{{/renderField}}
     {{/fields}}
```

Comment:
Do you even need this file? What are you overriding now?

Reply:
I have now removed the partials/important-message.html file as it is no longer needed.